xarray/core/formatting.py

```diff
@@ -176,6 +177,8 @@ def format_timedelta(t, timedelta_format=None):
 
 def format_item(x, timedelta_format=None, quote_strings=True):
     """Returns a succinct summary of an object as a string"""
+    if isinstance(x, PandasExtensionArray):
```

Comment:
Can we add `__str__` to PandasExtensionArray instead?

Reply:
Could you clarify what you mean? The point here is to bypass `PandasExtensionArray` here because its `__repr__` is `PandasExtensionArray(array=[...])`.